virocon/plotting.py

```diff
@@ -102,7 +184,25 @@ def plot_dependence_functions(model, model_desc=None, par_rename=None, axes=None
     return axes
 
 
-def plot_2D_isodensity(model, sample, model_desc=None, swap_axis=False, ax=None):
+def plot_2D_isodensity(model, sample, model_desc=None, swap_axis=False, ax=None):   
+    """
+    Function to plot the 2-dimensional isodensities of the indicated model.
+       
+    Parameters
+    ----------
+    model :  MultivariateModel
+        The model to be used to plot the isodensities.  
+    sample : ndarray of floats
+        The environmental data sample that should be plotted against the fit.
+    model_desc: dict, optional
+        Generated model description. Defaults to None.
```

Comment:
The description of the model. If None (the default) a generic model description will be used.